src/pages/Wallet/SendModalCheckTransaction.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { convertAlphToSet, formatAmountForDisplay } from 'alephium-js'
+import styled from 'styled-components'
+
+import InfoBox from '../../components/InfoBox'
+import { ModalFooterButton, ModalFooterButtons } from '../../components/Modal'
+import { SendTransactionData } from './SendModal'
+
+interface SendModalCheckTransactionProps {
+  data: SendTransactionData
+  fees: bigint
+  onSend: () => void
+  onCancel: () => void
+}
+
+const SendModalCheckTransaction = ({ data, fees, onSend, onCancel }: SendModalCheckTransactionProps) => {
```

Comment:
Could we maybe store those "step" pages in `pages/Wallet/SendModal/...` ? WDYT?

Reply:
Currently we are mixing components that are modals with components that are pages. For a while now I think that we should split `pages` and `modals`. Modals can be displayed on top of every page. Pages take hold of the whole visual area of the app (except top app bar and side bar). What would you think of the following structure?

```
- components
- pages
- modals
```

or

```
- components
  - modals
- pages
```

or even

```
- components
  - modals
  - pages
```

Reply:
So, i would say:

- A component is an functional item that can be reused in multiple places.
- Pages and modals are layouts made of multiple components.

I'm in favor of either the first solution, or:

```
- components
- layout
  - pages
  - modals 
```

Reply:
Ah, ok, so in your mental model, "components" are not "React components" but rather atomic, functional elements that are purpose is to be reused. I proposed option 3 with the "React components" meaning of "components" in mind. But since we have other categorization of React components (ex: `routes`, `contexts`), then I also agree that I am not in favor of option 2 and 3.

I would like to keep things simple and flat and avoid overengineering for the moment. Our project is still relatively small, our team is also small, and our goal is to save time during development by making things intuitive. I would like to go with option 1 for now and reconsider the categorization of our files if the need comes (like it did now 😄).

```
- components
- modals
- pages
```

In this case, and to answer your initial question, I'll move the send modal steps in: `/modals/SendModal/...`.

Reply:
Sounds good 👍

Reply:
Would you mind if I refactor this (as well as https://github.com/alephium/alephium-wallet/pull/128#discussion_r811194189)  in the next PR? It will cause quite some conflicts that will be annoying to solve.